app/shared/src/commonMain/kotlin/ui/subject/episode/video/settings/DanmakuRegexFilterGroup.kt

```diff
@@ -0,0 +1,372 @@
+                dismissButton = { TextButton({ showConfirmDelete = false }) { Text("取消") } },
+            )
+        }
+
+//        if (showEditFilter) {
+//            AddRegexFilterDialog(
+//                onDismissRequest = { showEditFilter = false },
+//                onConfirm = { name, regex ->
+//                    if (name.isBlank() || regex.isBlank()) {
+//                        errorMessage = "名字和正则不能为空"
+//                        showError = true
+//                    } else {
+//                        onEdit()
+//                        showEditFilter = false
+//                    }
+//                },
+//                title = { Text("编辑正则过滤器") },
+//                description = {
+//                    Text("请正确添加正则表达式，例：第一个字符为数字：'^[1-9]{1}\$'.\n符合正则的弹幕讲不会被显示")
+//                },
+//
+//                )
+//        }
+
+        Box {
+            var showMore by remember { mutableStateOf(false) }
+            DropdownMenu(
+                expanded = showMore,
+                onDismissRequest = { showMore = false },
+            ) {
+                DropdownMenuItem(
+                    leadingIcon = {
+                        if (item.isEnabled) {
+                            Icon(Icons.Rounded.VisibilityOff, null)
+                        } else {
+                            Icon(Icons.Rounded.Visibility, null)
+                        }
+                    },
+                    text = {
+                        if (item.isEnabled) {
+                            Text("禁用")
+                        } else {
+                            Text("启用")
+                        }
+                    },
+                    onClick = {
+                        onEnabledChange(!item.isEnabled)
+                        showMore = false
+                    }
+                )
+//                DropdownMenuItem(
+//                    leadingIcon = { Icon(Icons.Rounded.Edit, null) },
+//                    text = { Text("编辑") }, // 直接点击数据源一行也可以编辑, 但还是在这里放一个按钮以免有人不知道
+//                    onClick = {
+//                        showMore = false
+//                        onEdit()
+//                    }
+//                )
+                DropdownMenuItem(
+                    leadingIcon = { Icon(Icons.Rounded.Delete, null, tint = MaterialTheme.colorScheme.error) },
+                    text = { Text("删除", color = MaterialTheme.colorScheme.error) },
+                    onClick = {
+                        showMore = false
+                        showConfirmDelete = true
+                    }
+                )
+            }
+
+            IconButton({ showMore = true }) {
+                Icon(
+                    Icons.Rounded.MoreVert,
+                    contentDescription = "更多",
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun AddRegexFilterDialog(
+    onDismissRequest: () -> Unit,
+    onConfirm: (String, String) -> Unit, // onConfirm now accepts the text field value
```

Comment:
这两个 string 有什么意义, 可以写里面
```suggestion
    onConfirm: (name: String, regex: String) -> Unit, // onConfirm now accepts the text field value
```